### PR TITLE
Implement new blocksThemes concept

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

File: packages/volto-light-theme/src/components/Blocks/schema.js [hunk 1]
```diff
@@ -10,25 +10,24 @@ const messages = defineMessages({
...
 export const defaultStylingSchema = ({ schema, formData, intl }) => {
-  const colors =
-    config.blocks?.blocksConfig?.[formData['@type']]?.colors ||
-    config.settings.backgroundColors;
+  const themes =
+    config.blocks?.blocksConfig?.[formData['@type']]?.themes ||
+    config.blocks.blocksThemes;
 
   const defaultBGColor =
     config.blocks?.blocksConfig?.[formData['@type']]?.defaultBGColor ||
```
Comment (davisagli) [inline packages/volto-light-theme/src/components/Blocks/schema.js:18]: Change to `defaultTheme`?

File: packages/volto-light-theme/src/transforms/to6.ts [hunk 2]
```diff
@@ -15,7 +17,7 @@ function* visitBlocks(blocks) {
 
 export function migrateToVLT6ColorAndWidthModel(data: BlocksFormData) {
   const NORMALIZED_WIDTHS = [
-    ...config.settings.blockWidths,
+    ...config.blocks.blocksWidths,
```
Comment (davisagli) [inline packages/volto-light-theme/src/transforms/to6.ts:20]: config.blocks.widths

Comment (davisagli) [reply packages/volto-light-theme/src/transforms/to6.ts:20]: Never mind, I didn't see this is the transform.

Comment (davisagli) [reply packages/volto-light-theme/src/transforms/to6.ts:20]: Uh, but we aren't transforming the settings, so I think it is wrong...

File: packages/volto-light-theme/src/components/Blocks/schema.js (hunk 1 above)
Comment (sneridagh) [reply packages/volto-light-theme/src/components/Blocks/schema.js:18]: Done

File: packages/volto-light-theme/src/transforms/to6.ts (hunk 2 above)
Comment (sneridagh) [reply packages/volto-light-theme/src/transforms/to6.ts]: Fixed

**Comment by sneridagh** [comment]
Merging into vl6, there is some a11y failings, but there are present in there too.

**Pull request merged**